### ISSUE TITLE
T2-MI Support

### ISSUE THREAD

**Issue by Delitants** [catalinii/minisatip#1192]
Hi, does minisatip support T2-MI on transponders? If yes, how to tune such a transponder?

**Comment by catalinii** [comment]
T2m is not supported, @Yuri666 had some interest in it, but we did not complete the implementation

**Comment by Delitants** [comment]
> What is the use case for this? Is this about "Modulator interface"?

It’s a terrestrial signal on the satellite transponder.

**Comment by Jalle19** [comment]
@Delitants you may be interested in https://github.com/catalinii/minisatip/pull/1303

**Comment by Delitants** [comment]
> [@Delitants](https://github.com/Delitants) you may be interested in [#1303](https://github.com/catalinii/minisatip/pull/1303)

Hm, does that handle regular T2-MI Biss encrypted streams? Seems like some custom stuff.

**Comment by Jalle19** [comment]
It allows you to decrypt with BISS, then you can pass the stream to a T2-MI de-encapsulator

**Comment by Delitants** [comment]
> It allows you to decrypt with BISS, then you can pass the stream to a T2-MI de-encapsulator

Well, that's the thing, where is a de-encapsulator? I would expect to tune such a transponder, set the biss key via some flag to minisatip and have a ready decoded and de-encapsulated services to watch without extra software layered in between. Currently I have to use Astra Cesbo for this.

**Comment by Jalle19** [comment]
There's astra-sm and various tools on GitHub for de-encapsulating T2-MI

**Comment by Delitants** [comment]
> There's astra-sm and various tools on GitHub for de-encapsulating T2-MI

Yes and as I use one of them. Minisatip to do it itself was in question, but not critical.

**Comment by Jalle19** [comment]
Yeah. In my opinion it's out of scope for this project.
